system/service/LlmService.py

```diff
@@ -0,0 +1,26 @@
+from flask import current_app
+
+import openai
+
+class LlmService:
+    def __init__(self):
+        openai.api_key = os.getenv('OPENAI_API_KEY')
```

Comment:
missing import `os`

Reply:
solved in [a44f8c7](https://github.com/NYCU-CloudNativeTeam/DocumentSystem/pull/5/commits/a44f8c7da068aa9e21a01a385fffdd7867369420)